docs/api/createAction.mdx

```diff
@@ -103,21 +100,23 @@ const counterReducer = createReducer(0, (builder) => {
 })
 ```
 
+<!-- TODO: how do we handle this? -->
```

Comment:
:question: I dunno, what "this" are we referring to here? :)

Reply:
I don't think we have anything remaining that actually depends on the type being a string - but I'm assuming we still want to advise that they should be strings?